llvm/lib/Target/WebAssembly/WebAssembly.td

```diff
@@ -114,19 +122,20 @@ def : ProcessorModel<"mvp", NoSchedModel, []>;
 // consideration given to available support in relevant engines and tools, and
 // the importance of the features.
 def : ProcessorModel<"generic", NoSchedModel,
-                      [FeatureBulkMemory, FeatureMultivalue,
-                       FeatureMutableGlobals, FeatureNontrappingFPToInt,
-                       FeatureReferenceTypes, FeatureSignExt]>;
+                      [FeatureBulkMemory, FeatureBulkMemoryOpt,
+                       FeatureMultivalue, FeatureMutableGlobals,
+                       FeatureNontrappingFPToInt, FeatureReferenceTypes,
+                       FeatureCallIndirectOverlong, FeatureSignExt]>;
```

Comment:
```suggestion             
                       FeatureCallIndirectOverlong, FeatureMultivalue,
                       FeatureMutableGlobals, FeatureNontrappingFPToInt,
                       FeatureReferenceTypes, FeatureCallIndirectOverlong,
                       FeatureSignExt]>;
```

Reply:
The replacement here has `FeatureCallIndirectOverlong` listed twice.

Reply:
Sorry I meant to move it to the beginning (to maintain the sorted order)